tests/extra/test_job_management.py

```diff
@@ -455,7 +447,65 @@ def start_job(row, connection_provider, connection, **kwargs):
         assert set(result.status) == {"running"}
         assert set(result.backend_name) == {"foo"}
 
-
+    @mock.patch('openeo.extra.job_management.datetime', autospec=True)
+    @mock.patch('openeo.extra.job_management.timezone', autospec=True)
```

Comment:
as mentioned higher, I would leverage the "time_machine" test helper/fixture instead of these convoluted datetime mocks

e.g. see pytest-specific docs at https://github.com/adamchainz/time-machine?tab=readme-ov-file#pytest-plugin